extensions/windows-event-log/wel/JSONUtils.h

```diff
@@ -20,9 +20,10 @@
 #undef RAPIDJSON_ASSERT
 #define RAPIDJSON_ASSERT(x) if (!(x)) throw std::logic_error("rapidjson exception");  // NOLINT
 
-#include <pugixml.hpp>
-
 #include <stdexcept>  // for std::logic_error
+#include <string>
```

Comment:
`<stdexcept>` can go to the implementation file, it's not used in the header.

Reply:
thanks, fixed in [95aeacd](https://github.com/martinzink/nifi-minifi-cpp/commit/95aeacd78262361de87609058f1f208b01e4fe3b)

Reply:
It turns out its needed in the header, because in line 21 RAPIDJSON_ASSERT is redefined as std::logic so the rapidjson include needs the stdexcept